### PR TITLE
Feat/sprint3

### PR DESCRIPTION
**Pull request by PaulBOURGEAIS**

### CODE REVIEW

File: src/ui/components/HourChecker/HourChecker.tsx [hunk 1]
```diff
@@ -136,7 +138,11 @@ const HourChecker = memo((props: HourCheckerProps) => {
     );
 });
 
-const useStyles = makeStyles<{ width: string }>({ "name": { HourChecker } })((theme, { width }) => ({
+const useStyles = makeStylesEdt<{ width: string }>({ "name": { HourChecker } })((theme, { width }) => ({
```
Comment (MichaelC67) [inline src/ui/components/HourChecker/HourChecker.tsx:141]: Could (re)explain me with sometimes you use const useStyles for CSS and sometimes the sx attribute ? ?

File: src/ui/components/Planner/DayOverview/DayOverview.tsx [hunk 2]
```diff
@@ -0,0 +1,202 @@
...
+    };
+
+    const renderRow = (h: TimeLineRowType): any => {
+        return (
+            <Box className={classes.rowContainer} key={uuidv4()}>
+                <Box className={classes.rowLabel}>
+                    <Typography className={classes.hourLabel}>{h.label}</Typography>
+                </Box>
+
+                <HourChecker responses={h.options} value={h.value}></HourChecker>
```
Comment (MichaelC67) [inline src/ui/components/Planner/DayOverview/DayOverview.tsx:154]: `<HourChecker responses={h.options} value={h.value} />` possible ?

File: src/ui/components/Planner/DayPlanner/DayPlanner.tsx [hunk 3]
```diff
@@ -0,0 +1,155 @@
...
+    const buttonsOnClick = (): void => {
+        setDisplayDayOverview(true);
+        setDayOverviewSelectedDate(date);
+    };
+
+    const renderBottomPart = () => {
+        return dayRelativeTime === -1 ? (
+            <Box className={classes.textBox}>
+                <Typography className={classes.workTimeText}>
+                    Durée totale travaillée : <b>{workedHoursSum}</b> minutes
```
Comment (MichaelC67) [inline src/ui/components/Planner/DayPlanner/DayPlanner.tsx:68]: Text as variable ? 
`<b>` should not be used.

File: src/ui/components/Planner/DayPlanner/DayPlanner.tsx [hunk 4]
```diff
@@ -0,0 +1,155 @@
...
+                </Typography>
+            </Box>
+        ) : dayRelativeTime === 0 || workedHoursSum !== 0 ? (
+            <Box className={classes.buttonBox}>
+                <Button className={classes.button} onClick={buttonsOnClick}>
+                    Continuer
```
Comment (MichaelC67) [inline src/ui/components/Planner/DayPlanner/DayPlanner.tsx:74]: Text as variable ?

File: src/ui/components/Planner/DayPlanner/DayPlanner.tsx [hunk 5]
```diff
@@ -0,0 +1,155 @@
...
+                </Button>
+            </Box>
+        ) : (
+            <Box className={classes.buttonBox}>
+                <Button className={cx(classes.button, classes.buttonFuture)} onClick={buttonsOnClick}>
+                    Commencer
```
Comment (MichaelC67) [inline src/ui/components/Planner/DayPlanner/DayPlanner.tsx:80]: Text as variable ?

File: src/ui/theme/make-style-edt.ts [hunk 6]
```diff
@@ -0,0 +1,44 @@
+import { makeStyles } from "tss-react/mui";
+import { Css, CSSObject, Cx } from "tss-react/types";
+import { EdtTheme } from "./edt-theme";
+
+type MakeStylesParams =
```
Comment (MichaelC67) [inline src/ui/theme/make-style-edt.ts:5]: Documentation might be needed about that

File: src/ui/theme/theme.tsx [hunk 7]
```diff
@@ -14,15 +16,19 @@ const theme = createTheme({
             light: "#2C70DE",
         },
         secondary: {
-            main: "#f50057",
+            main: "#1f4076",
```
Comment (MichaelC67) [inline src/ui/theme/theme.tsx:19]: We perhaps could have a rule for the notation #xxxxxx =>  Uppercase or LowerCase ?

**Review by MichaelC67 — CHANGES_REQUESTED** [review]
Few comments + Pipeline needs to be fixed. Thanks !

File: src/ui/components/HourChecker/HourChecker.tsx (hunk 1 above)
Comment (PaulBOURGEAIS) [reply src/ui/components/HourChecker/HourChecker.tsx:139]: It does the same thing. Sometimes the syntax of useStyles is heavy for just one prop (an import, and 2 more const). For example, when we only need to set the prop display: "none" or "visible" deppending on a bool, we find the useStyle syntax heavier so we use sx. When there is more than 1 prop to edit at once, we use useStyle. 
I fixed all other encountered cases (sx with more than 1 prop) by using useStyle.

**Review by MichaelC67 — APPROVED** [review]

**Review by MichaelC67 — CHANGES_REQUESTED** [review]
Thanks for fixing the build process.
Could you have a look at the SonarCloud Analysis 
=> https://sonarcloud.io/project/issues?resolved=false&severities=CRITICAL&sinceLeakPeriod=true&types=CODE_SMELL&pullRequest=21&id=InseeFrLab_lunatic-edt 
There is a critical Code Smell (it would be nice to fix also the Majors).

**Review by MichaelC67 — APPROVED** [review]

**Pull request merged**